### ISSUE TITLE
Unable to download dataset (DOI: 10.5061/dryad.gn001)

### ISSUE THREAD

**Issue by jleighherzog** [CDL-Dryad/dryad-product-roadmap#1280]
**Bug description:**
User has reported that the dataset for this published submission cannot be downloaded: 

_"This is to inform, that the dataset https://datadryad.org/stash/dataset/doi:10.5061/dryad.gn001 cannot be downloaded via API (due to a technical error?)."_

I have attempted to download the files myself via the "Download Dataset" feature and I also receive an error shortly into the download process. 

Title: Data from: An ab initio electronic transport database for inorganic materials
DOI: 10.5061/dryad.gn001
Published: June 18, 2018

**Steps to reproduce:**
Select "Download Dataset"

**Expected behavior:**
When selected, all files should begin downloading. Download progress should not timeout or fail.

**Comment by sfisher** [comment]
It will take a very long time for the download to assemble all the pieces.  If the user wants it faster and not to wait for a large object to be assembled and zipped then they can download each of the files individually from the Ui without needing to wait.

However, I tested this out and it downloads correctly for me.

This first is a screenshot of the download completing (100%).  It takes a very long time to download since this is a very large file that is actually over our standard size limit.  For me it was 2.5 hours to download on our faster server connection.  I'm not going to try it from home since it will probably take a lot longer like a day.

![Screen Shot 2021-06-01 at 12 25 03 PM](https://user-images.githubusercontent.com/105433/120396396-57bc6680-c2eb-11eb-9992-b31044703cba.png)

I also extracted all the files form the zip file and they all extracted correctly.  This is a screenshot of that.

![Screen Shot 2021-06-01 at 2 54 51 PM](https://user-images.githubusercontent.com/105433/120396803-08c30100-c2ec-11eb-9edf-e890a305c4f9.png)

So IDK what is going wrong for this user.  There are plenty of opportunities for bad internet connections or other things to cut off the download for a file this big.  We are also serving the downloads directly from Amazon S3 which should be very, very reliable and we are not relying on anything special in our code to download.  It's just 1) download is prepared, 2) we give them a presigned url and 3) send them to Amazon's Simple Storage Service (S3) for the download.
